### DISCUSSION TITLE
Welcome to kuliahkaryawanK2 Discussions!

### DISCUSSION THREAD

**Discussion by kuliahkaryawanK2** [kuliahkaryawanK2/kuliahkaryawanK2#5] [Announcements]
Q&A Seputar Web Programing

**Comment by abdillahgfr** [comment]


**Reply by Marsudii** [reply]
Untuk Bahasa Pemerograman yang dipakai saat ini PHP dengan database Maria DB

**Reply by abdillahgfr** [reply]
> Untuk Bahasa Pemerograman yang dipakai saat ini PHP dengan database Maria DB

kalo untuk PHP nya ada minimal versi berapa kak?

**Reply by Marsudii** [reply]
Untuk saat ini versi PHP yang digunakan PHP 7.4 dan PHP 8.1

**Reply by abdillahgfr** [reply]
> Untuk saat ini versi PHP yang digunakan PHP 7.4 dan PHP 8.1

oke terimakasih kak
